src/ex6/client/src/components/todo-add/todoAdd.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState,useCallback } from "react";
+import PropTypes from "prop-types";
+import ItemClient from "../../services/item_client.js";
+import { Button } from "monday-ui-react-core";
+import "./todoAdd.css";
+
+const TodoAdd = ({setTodoList,}) => {
+    const [todoInput, setTodoInput] = useState('');
+    const [loading, setLoading] = useState(false);
+    const itemClient = new ItemClient();
+
+    const onClick = useCallback((flag) => {

```

Comment:
- useCallback is not being used correctly here - as setLoading (the dependency) never changes
 - it add complexity to this component - I think it's better overall to skip it